build.py

```diff
@@ -267,6 +267,11 @@ def build_package(target):
             if target == 'boost':
                 # using boostorg namespace instead of forking 50+ relatively linked submodules
                 run_cmd(['git', 'clone', 'https://github.com/boostorg/boost'])
+            elif target == 'avro':
+                run_cmd(['git', 'clone', 'https://github.com/{0}/{1}'.format('apache', target)])
+            # TODO: Remove me
+            elif target == 'mungefs':
```

Comment:
Need to remove this upon merging of irods/mungefs#15.

Reply:
merged